.gitignore

```diff
@@ -20,5 +20,6 @@ faces-config.NavData
 *.lock.db
 **/overlays
 **/node_modules
+camunda-h2-dbs
```

Comment:
I added this, so that even if we create a `camunda-h2-dbs` directory, we don't commit it by accident.

Reply:
👍